src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -60,7 +60,7 @@ export class MemberListComponent implements OnChanges, OnDestroy {
 
   @Input() group? : Group;
 
-  state: 'error' | 'ready' | 'fetching' | 'processing' = 'fetching';
+  state: 'error' | 'ready' | 'fetching' = 'fetching';
```

Comment:
is there a reason not to use fetchstate?

Reply:
With the fetchstate version, the table is empty during loading, and the data isn't stored anymore in the component, but it's needed for selection.